extensions/eclipse-che-theia-terminal/package.json

```diff
@@ -16,15 +16,15 @@
     "src"
   ],
   "dependencies": {
-    "@theia/core": "0.3.17",
-    "@theia/terminal": "0.3.17",
+    "@theia/core": "0.3.19",
+    "@theia/terminal": "0.3.19",
     "xterm": "3.9.1",
     "@eclipse-che/workspace-client": "latest",
     "@eclipse-che/api": "latest"
   },
   "license": "EPL-2.0",
   "devDependencies": {
-    "typescript": "2.9.2",
+    "typescript": "^3.1.3",
```

Comment:
maybe remove carret (well that's fine as we got CQ on dev dependencies for a version and later but maybe best to stick with one version here)